### PR TITLE
Add Basic Launch.JSON

### PR DESCRIPTION
**Pull request by say25**
**What does this pull request introduce? Please describe**  
Adds basic VSCode Launch.Json - #159

**How did you verify that your changes work as expected? Please describe**  
Please describe your methods of testing your changes.

**Example**  
Please describe how we can try out your changes  
1. Create a new direflow app
2. Build with NPM or Yarn
3. Open with VSCode
4. Use built in debugger attacher

### CODE REVIEW

File: templates/js/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "command": "yarn start:no-browser",
```
Comment (say25) [inline templates/js/.vscode/launch.json:8]: I left this as yarn but part of me thinks, should direflow take in yarn or npm as a command and use that here

**Comment by Silind** [comment]
Awesome, @say25 ! :smile: 
Thanks a lot for adding this!

I created a feature branch for this.
We need a little more configuration from the CLI in order to have this working optimally.
As you mention, we need to ask people if they're using npm or yarn when creating a new project.
We also need to ask them if they want the launch.json file. Some may use webstorm or atom, thus it doesn't make much sense for them to have it.

So the CLI needs to be extended to ask these two questions during the setup of a new project.
If you're up for the challenge, you're very welcome to implement that as well :smile: 

If not, I'll merge this into the feature branch I created, and we'll allocate time to finish it.

**Comment by say25** [comment]
@Silind I think you might be better suited to take the user experience you have in mind to completion.

**Comment by Silind** [comment]
That's totally cool!
Thanks a lot for your contribution :blush:

**Pull request merged**